src/workerd/api/container.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2025 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#pragma once
+// APIs that an Actor (Durable Object) uses to access its own state.
+//
+// See actor.h for APIs used by other Workers to talk to Actors.
+
+#include <workerd/io/container.capnp.h>
+#include <workerd/io/io-own.h>
+#include <workerd/jsg/jsg.h>
+
+namespace workerd::api {
+
+class Fetcher;
+
+class Container: public jsg::Object {
```

Comment:
Can you add code comments explaining what a Container is

Reply:
543bac342c20abe59b546cf1fd8e2585cad9a8c9